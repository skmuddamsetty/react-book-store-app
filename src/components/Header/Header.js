import React from 'react';
import { NavLink } from 'react-router-dom';

const Header = () => (
  <header>
    <h1>Book Store</h1>
    <NavLink to='/' activeClassName='is-active' exact={true}>
      Dashboard
    </NavLink>
    <NavLink to='/add-book' activeClassName='is-active' exact={true}>
      Add Book
    </NavLink>
  </header>
);

export default Header;
