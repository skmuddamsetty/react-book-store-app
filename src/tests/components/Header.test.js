import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Header from '../../components/Header/Header';
test('should render header correctly', () => {
  const wrapper = shallow(<Header />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
