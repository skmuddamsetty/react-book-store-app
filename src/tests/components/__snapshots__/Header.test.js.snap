// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<header
  className="header"
>
  <ul
    className="nav nav-pills nav-justified"
  >
    <li
      className="nav-item"
    >
      <h4>
        Book Store
      </h4>
    </li>
    <li
      class="nav-item"
    >
      <[object Object]
        activeClassName="is-active"
        className="nav-link"
        exact={true}
        to="/"
      >
        Dashboard
      </[object Object]>
    </li>
    <li
      class="nav-item"
    >
      <[object Object]
        activeClassName="is-active"
        className="nav-link"
        exact={true}
        to="/add-book"
      >
        Add Book
      </[object Object]>
    </li>
  </ul>
</header>
`;
